### ISSUE TITLE
The code encountered many difficulties during execution.

### ISSUE THREAD

**Issue by ACDA-cell** [qukehua/KSOF#1]
First of all, I would like to thank you for your open-source contribution. However, I encountered many difficulties when running the code you provided. The code is poorly written, and I found several basic errors, such as incorrect variable names. I have a key question regarding line 292 in `main_h36m.py`: `mpjpe_p3d_h36 = torch.mean(torch.norm(p3d_h36[:, in_n:] - p3d_out_4, dim=3))` and in the testing phase: `mpjpe_p3d_h36 = torch.sum(torch.mean(torch.norm(p3d_h36[:, in_n:] - p3d_out_4, dim=3), dim=2), dim=0)`. Here, `p3d_h36[:, in_n:]` represents the ground truth, and `p3d_out_4` is also the future ground truth, so clearly, these two tensors are the same, not the difference between the ground truth and the predicted values. I am confused about this. If you could clarify this for me, I would greatly appreciate it!

**Comment by qiaogh97** [comment]
I guess the author trained the model on another code repo, and this repo was never run completely.
